ccd-definition/AuthorisationCaseField/staffLRspec.json

```diff
@@ -136,5 +136,17 @@
     "CaseFieldID": "additionalInformationForJudge",
     "UserRole": "caseworker-civil-staff",
     "CRUD": "R"
+  },
+  {
```

Comment:
Please can you remove this and see if it works. Not sure if we need this.

Reply:
code works without any entry in staffLRspec.json